book.rb

```diff
@@ -0,0 +1,10 @@
+class Book
+  attr_accessor :title, :author
+  attr_reader :rentals
+
+  def initialize(title: 'unknown', author: 'unknown', rentals: [])
+    @title = title
+    @author = author
+    @rentals = rentals
+  end
+end
```

Comment:
- [ ] It would be nice if you have a method to create rentals to form the many-to-many relationship 👍 the method should look like below ⬇️

```ruby
  def add_rental(person, date)
    # *the book itself should be sent as a parameter to create the new rental achieving the 'has-many' association. 
    Rental.new(date, self, person)
  end

```